<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>valores_window</class>
 <widget class="QWidget" name="valores_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Valores - PrePRODIC3D</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>644</width>
        <height>742</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Valores</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Valores generales del dominio por variable</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QListWidget" name="lw_variables">
                  <item>
                   <property name="text">
                    <string>Temperatura</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string/>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="le_general_value"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Conductividad</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="le_k"/>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Extrapola excepto</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QGroupBox" name="groupBox_11">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QCheckBox" name="cb_ipun">
                     <property name="layoutDirection">
                      <enum>Qt::RightToLeft</enum>
                     </property>
                     <property name="text">
                      <string>Esquinas</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QGroupBox" name="groupBox_10">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_4">
                   <item row="1" column="0">
                    <widget class="QCheckBox" name="cb_iborx">
                     <property name="layoutDirection">
                      <enum>Qt::RightToLeft</enum>
                     </property>
                     <property name="text">
                      <string>X</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QCheckBox" name="cb_iborz">
                     <property name="layoutDirection">
                      <enum>Qt::RightToLeft</enum>
                     </property>
                     <property name="text">
                      <string>Z</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QCheckBox" name="cb_ibory">
                     <property name="layoutDirection">
                      <enum>Qt::RightToLeft</enum>
                     </property>
                     <property name="text">
                      <string>Y</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Valor general</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Flotabilidad</string>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Coef. exp. term.</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_3"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Gravedad</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEdit_4"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Ángulo</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEdit_5"/>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox">
                  <property name="text">
                   <string>Activar flotabilidad</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chb_local_value">
            <property name="text">
             <string>Para aplicar valores locales para alguna variable, active la siguiente casilla</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Valores locales por variable</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_7">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="title">
                <string>Regiones</string>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <item row="2" column="0" colspan="2">
                 <widget class="QPushButton" name="pb_add_region">
                  <property name="text">
                   <string>Agregar una región</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QPushButton" name="pb_remove_region">
                  <property name="text">
                   <string>Eliminar última región</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QListWidget" name="lw_regions">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <item>
                   <property name="text">
                    <string>Region 1</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="10" column="0">
                 <widget class="QLineEdit" name="le_local_value"/>
                </item>
                <item row="10" column="1">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Valor local</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="11" column="0">
                 <widget class="QLineEdit" name="lineEdit_7"/>
                </item>
                <item row="11" column="1">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Sc</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="14" column="0">
                 <widget class="QLineEdit" name="lineEdit_8"/>
                </item>
                <item row="14" column="1">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Sp</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="17" column="0">
                 <widget class="QLineEdit" name="le_local_k"/>
                </item>
                <item row="17" column="1">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>k local</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QCheckBox" name="chb_fixed_value">
                  <property name="text">
                   <string>Valor fijo</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QCheckBox" name="chb_linear_source">
                  <property name="text">
                   <string>Fuente lineal</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>Conformación de la región</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QListWidget" name="lw_volumes">
                  <item>
                   <property name="text">
                    <string>Volumen 1</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_add_volume">
                  <property name="text">
                   <string>Agregar un volumen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_remove_volume">
                  <property name="text">
                   <string>Eliminar último volumen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_9">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>Longitud</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="2">
                    <widget class="QLineEdit" name="le_y_lon">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="le_x_start">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QLineEdit" name="le_y_start">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="le_x_lon">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>X</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>Y</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Inicio</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QLabel" name="label_14">
                     <property name="text">
                      <string>Z</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="3">
                    <widget class="QLineEdit" name="le_z_start">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="3">
                    <widget class="QLineEdit" name="le_z_lon">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
